truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(isEmployed, isOnVacation));
+
+// 2
+function closeCompare(numberA, numberB, numbersMargin){
+  if (numbersMargin === undefined) numbersMargin = 0;
+  const numbersDiff = Math.abs(numberA - numberB);
+  if (numbersDiff <= numbersMargin) return 0
+  if (numberA < numberB) return -1
+  return 1
+}
+
+// 3
+const or = (a, b) => {
+  if (a == true) return true
+  if (b == true) return true
+  return false
+};
+
+const xor = (a, b) => {
+  if (a == true) {
+    if (b === true) return false
+    else return true
+  }
+  if (b == true) {
+    if (a === true) return false
+    else return true
+  }
+  return false
+};
+
+// 4
+function isTruthy(bool, function1, function2) {
+  if (bool == true) return function1();
+  return function2()
+}
+
+// 5
+function rentalCarCost(days) {
+  if (days >= 3 && days <= 6) {
+    return days * 40 - 20
+  }
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+  return days * 40
+}
+
+// 6
+function getRealFloor(floorNumber) {
+  if (floorNumber >= 13) return floorNumber - 2
+  if (floorNumber < 13 && floorNumber >= 1) return floorNumber -1;
+  return floorNumber
+}
+
+// 7
+function rpsls(pl1, pl2){
+  if ((pl1 === "rock" && pl2 === "lizard") || (pl1 === "rock" && pl2 === "scissors")) return 'Player 1 Won!'
+  if ((pl1 === "scissors" && pl2 === "paper") || (pl1 === "scissors" && pl2 === "lizard")) return 'Player 1 Won!'
+  if ((pl1 === "paper" && pl2 === "rock") || (pl1 === "paper" && pl2 === "spock")) return 'Player 1 Won!'
+  if ((pl1 === "lizard" && pl2 === "spock") || (pl1 === "lizard" && pl2 === "paper")) return 'Player 1 Won!'
+  if ((pl1 === "spock" && pl2 === "scissors") || (pl1 === "spock" && pl2 === "rock")) return 'Player 1 Won!'
+
+  if ((pl2 === "rock" && pl1 === "lizard") || (pl2 === "rock" && pl1 === "scissors")) return 'Player 2 Won!'
+  if ((pl2 === "scissors" && pl1 === "paper") || (pl2 === "scissors" && pl1 === "lizard")) return 'Player 2 Won!'
+  if ((pl2 === "paper" && pl1 === "rock") || (pl2 === "paper" && pl1 === "spock")) return 'Player 2 Won!'
+  if ((pl2 === "lizard" && pl1 === "spock") || (pl2 === "lizard" && pl1 === "paper")) return 'Player 2 Won!'
+  if ((pl2 === "spock" && pl1 === "scissors") || (pl2 === "spock" && pl1 === "rock")) return 'Player 2 Won!'
+
+  if (pl1 === pl2) return "Draw!"
+}
+
+// 8
+function isValidPassword(password) {
+  return password === 'StrongPassword123'
+}
+console.log(isValidPassword('password'))
+
+// 9
+function isPasswordStrong(passwordVar) {
```

Comment:
Split this big problem into multiple steps in a similar way we broke down the `getCustomPizzaCost` function